### PR TITLE
Config Order Maintained

### PR DESCRIPTION
**Pull request by mexi-cano**
What (if anything) did you refactor?
- Updated config model and serialization to maintain object order. Updated init config data to correct order.

### CODE REVIEW

**Comment by CalebPena** [comment]
I think that you need to run `python manage.py makemigrations` and than migrate.

**Comment by mexi-cano** [comment]
Using an `OrderedDict` to return the data field as an OrderedDict instance did not work for me. I tried returning the ordered data as such but the approach did not work.

```
def ordered_data(self):
        return json.loads(json.dumps(self.data), object_pairs_hook=OrderedDict)
```

Instead I created a custom OrderedJSONField that inherits from JSONField and overrides the serialization and deserialization methods to handle ordered dictionaries. To achieve this, I made changes to the model, serializer, and created a custom field class:

Custom OrderedJSONField (fields.py):

I created a custom OrderedJSONField class that inherits from Django's `JSONField`.
In the `from_db_value` and `to_python methods`, I use `json.loads` with a `custom object_pairs_hook` that converts the deserialized JSON object into an `OrderedDict`. This ensures that the order of the keys is preserved when retrieving data from the database.
The `get_prep_value` method is used to serialize the JSON data before saving it to the database.


Model (models.py):

In the model definition, I replaced the default `JSONField` with our custom `OrderedJSONField` for the data field.
This ensures that the JSON data stored in the data field will maintain the order of the keys.


Serializer (serializers.py):

In the serializer, I specified the data field as an instance of `OrderedJSONField`.
This ensures that the serializer uses the custom field for serialization and deserialization of the JSON data.

@CalebPena

**Review by CalebPena — CHANGES_REQUESTED** [review]
Looks good, but you forgot to run `python manage.py makemigrations`.

**Review by CalebPena — APPROVED** [review]

**Comment by CalebPena** [comment]
@mexi-cano Could you fix the merge conflicts?

File: configuration/management/commands/add_config.py [hunk 1]
```diff
@@ -9,33 +9,31 @@ class Command(BaseCommand):
...
...RentHeatCreditRebate","_default_message":"Cash to pay property tax, rent, and heat bills (Colorado Property Tax/Rent/Heat Credit Rebate)"}},"category_name":"Housing and Utilities"},"transportation":{"benefits":{"rtdlive":{"_label":"transportationBenefits.rtdlive","_default_message":"Discounted RTD fares (RTD LiVE)"}},"category_name":"Transportation"},"healthCare":{"benefits":{"dentallowincseniors":{"_label":"healthCareBenefits.dentallowincseniors","_default_message":"Low-cost dental care for people 60 years of age or older (Colorado Dental Health Program for Low-Income Seniors)"}},"category_name":"Health Care"},"taxCredits":{"benefits":{"eitc":{"_label":"taxCreditBenefits.eitc","_default_message":"Federal tax credit: earned income (Earned Income Tax Credit)"},"ctc":{"_label":"taxCreditBenefits.ctc","_default_message":"Federal tax credit: child tax credit (Child Tax Credit)"},"coeitc":{"_label":"taxCreditBenefits.coeitc","_default_message":"State tax credit: earned income (Colorado Earned Income Tax Credit/Expanded Earned Income Tax Credit)"},"coctc":{"_label":"taxCreditBenefits.coctc","_default_message":"State tax credit: Colorado child tax credit"}},"category_name":"Tax Credits"}}
```
Comment (CalebPena) [inline configuration/management/commands/add_config.py:36]: Just realized that the `category_name` needs to be a formatted message.

**Review by CalebPena — CHANGES_REQUESTED** [review]

**Review by CalebPena — APPROVED** [review]

**Pull request merged**